packages/nextjs/hooks/scaffold-eth/useScaffoldWriteContract.ts

```diff
@@ -22,16 +23,19 @@ import {
  */
 export const useScaffoldWriteContract = <TContractName extends ContractName>(
   contractName: TContractName,
+  chain?: Chain,
```

Comment:
Thank you for pr @Hotmanics !

Some thoughts regarding params:
so if I want for example pass `writeContractParams` without `chain`, I need to use `undefined` every time?
```
const { writeContractAsync } = useScaffoldWriteContract("YourContract", undefined, {});
```
changing order to `contractName, writeContractParams?, chain?` will cause same problem when needed to pass `chain` without `writeContractParams`.

So maybe we need to change params to object `{contractName, writeContractParams?, chain?}` here and for all other hooks without object in params? And params will become more consistent. Because for now somewhere we have objects, somewhere not. Or another option: (contractName, options?: {writeContractParams?, chain?}) but I'm voting for one object

Don't change it right now, let's wait Shiv or Carlos

cc @technophile-04 @carletex

Reply:
You are absolutely correct. This was an oversight by me.

Not sure as the best path forward, I'm sure others will be able to provide a more detailed observation.

Appreciate you looking at the PR!